src/index.html

```diff
@@ -1,13 +1,489 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;0,700;1,500&display=swap"
+    rel="stylesheet">
+    <title>My Bike</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./image/favicon.svg"
+    >
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+
+            <a href="#home">
+              <img
+                src="./image/MyBiKE.svg"
+                alt="My Bike"
+                class="top-bar__logo-link"
+              >
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                href="+1 234 555-55-55"
+                class="top-bar__icons-phone">
```

Comment:
Use valid format href for telephone link
<a href="tel:123-456-7890">123-456-7890</a>